### PR TITLE
update template

### LINKED ISSUE #436: "Access key added" doesn't describe which key is added
When I view transaction which added / removed key – it is interesting which key was removed/added and with which permissions.

It is currently not visible https://explorer.near.org/transactions/DmAncJcrxCt4mPFmbmvdc8E3RfQbhTdSisoJ5Mmyzaok

### PR DESCRIPTION
**Pull request by icerove**
fix #436

### CODE REVIEW

File: frontend/src/components/transactions/ActionMessage.tsx [hunk 1]
```diff
@@ -123,18 +123,29 @@ const transactionMessageRenderers: TransactionMessageRenderers = {
...
+          </p>
         </>
       ) : (
         <>
           {"New key added for "}
           <AccountLink accountId={receiverId} />
           {`: ${actionArgs.public_key.substring(0, 15)}...`}
+          <p>
+            {`with permission ${actionArgs.access_key.permission}`}
+            {` and nounce ${actionArgs.access_key.nonce}`}
```
Comment (frol) [inline frontend/src/components/transactions/ActionMessage.tsx:147]:
```suggestion
            {`with permission ${actionArgs.access_key.permission} and nonce ${actionArgs.access_key.nonce}`}
```

File: frontend/src/components/transactions/__tests__/__snapshots__/ActionMessage.test.tsx.snap [hunk 2]
```diff
@@ -12,6 +12,10 @@ Array [
     receiver.test
   </a>,
   ": ed25519:BgXFiJS...",
+  <p>
+    with permission undefined
+     and nounce undefined
```
Comment (frol) [inline frontend/src/components/transactions/__tests__/__snapshots__/ActionMessage.test.tsx.snap:17]: This is either a bad test or bad implementation. Please, make sure we don't render/test undefined.

File: frontend/src/components/transactions/ActionMessage.tsx [hunk 3]
```diff
@@ -123,18 +123,29 @@ const transactionMessageRenderers: TransactionMessageRenderers = {
...
             accountId={
               actionArgs.access_key.permission.FunctionCall.receiver_id
             }
           />
+          {`: ${actionArgs.public_key.substring(0, 15)}...`}
+          <p>
+            {`with function call [${actionArgs.access_key.permission.FunctionCall.method_names.join(
+              ","
+            )}]`}
+            {` and nounce ${actionArgs.access_key.nonce}`}
```
Comment (frol) [inline frontend/src/components/transactions/ActionMessage.tsx:137]:
```suggestion
            {`with permission to call [${actionArgs.access_key.permission.FunctionCall.method_names.join(
              ","
            )}] methods and nonce ${actionArgs.access_key.nonce}`}
```

**Review by frol — APPROVED** [review]
Feel free to merge once you address the comments

File: frontend/src/components/transactions/__tests__/__snapshots__/ActionMessage.test.tsx.snap (hunk 2 above)
Comment (icerove) [reply frontend/src/components/transactions/__tests__/__snapshots__/ActionMessage.test.tsx.snap:17]: yup, I will update the test

**Pull request merged**